impl/config/config.go

```diff
@@ -62,6 +62,11 @@ type PKARRServiceConfig struct {
 	CacheSizeLimitMB int    `toml:"cache_size_limit_mb"`
 }
 
+type LogConfig struct {
```

Comment:
another way I handled this in the ssi service is by defining an enum of environments with preconfigured levels and we could couple other config to it. I think this gives us more flexibility so I'm in favor of it